src/Acquisition.jl

```diff
@@ -29,6 +30,25 @@ struct AcquisitionResults{S<:AbstractGNSS,T}
     }
 end
 
+function Base.show(io::IO, ::MIME"text/plain", acq_channels::Vector{Acquisition.AcquisitionResults{T1,T2}}) where {T1,T2}
+    header = ["PRN"; "CN0"; "Carrier doppler (Hz)"; "Code phase (chips)"]
```

Comment:
Oh and we should add the units here:
```suggestion
    header = ["PRN"; "CN0 (dBHz)"; "Carrier Doppler (Hz)"; "Code phase (chips)"]
```